From 676f5d4009c4dce6996689fc7663c411c08a7197 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristian=20Oth=C3=B3n=20Mart=C3=ADnez=20Vera?=
 <cfuga@cfuga.mx>
Date: Tue, 27 Jun 2023 12:36:33 -0600
Subject: [PATCH] l64seek.h: use off_t instead of loff_t
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

musl libc doesn't define loff_t, but nowadays you could use off_t for 64-bit offsets

Bug: https://bugs.gentoo.org/715842
Signed-off-by: Cristian Othón Martínez Vera <cfuga@cfuga.mx>
---
 src/l64seek.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/l64seek.h b/src/l64seek.h
index 5e8dc4b..cb7969a 100644
--- a/src/l64seek.h
+++ b/src/l64seek.h
@@ -27,7 +27,7 @@
  * offsets.
  */
 
-typedef loff_t off64_t;
+typedef off_t off64_t;
 typedef off64_t s64_t;
 
 off64_t l64seek(int fd, off64_t offset, int whence);
